cypress-tests/cypress/support/commands/general.ts

```diff
@@ -24,6 +24,10 @@ function getDataCy(value: string, exist?: boolean) {
   return cy.get(`[data-cy=${value}]`);
 }
 
+export function getRandomString(length: number=6) {
+  return Math.random().toString(20).substr(2, length)
+}
```

Comment:
Just nitpick details: no need to specify the type when it's automatically inferred by the default value. Also, `.substr` 
is deprecated ([ref](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr)), better using `.substring`.

```suggestion
export function getRandomString(length = 6) {
  return Math.random().toString(20).substring(2, length)
}
```

Reply:
You could also set the default to `8` and avoid passing the argument in the other files since you always use 8 -- this is ofc totally discretionary, feel free to keep it as is :smile: